BraintreeCore/src/main/java/com/braintreepayments/api/PostalAddress.kt

```diff
@@ -0,0 +1,78 @@
+package com.braintreepayments.api
+
+import android.os.Parcelable
+import android.text.TextUtils
+import android.os.Parcel
+
+// NEXT_MAYOR_VERSION: transform to data class and check for @Parcelize annotation
+/**
+ * Java object representing a postal address
+ */
+open class PostalAddress : Parcelable {
+    var recipientName: String? = null
```

Comment:
We may need to mark these properties as open for now to avoid a breaking change.
```suggestion
    open var recipientName: String? = null
```